Source/WarpX.H

```diff
@@ -341,6 +342,16 @@ public:
      */
     IntervalsParser get_load_balance_intervals () const {return load_balance_intervals;}
 
+    /**
+     * \brief Private function for spectral solver
+     * Applies a damping factor in the guards cells the extend
```

Comment:
Is this maybe a typo?
```suggestion
     * Applies a damping factor in the guards cells that extend
```

Reply:
Thanks for fixing the the typo.